runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -517,6 +517,13 @@ endif
 </#if>
 
 <#if uma.spec.processor.ppc && !uma.spec.type.aix>
+
+<#if uma.spec.flags.env_littleEndian.enabled && uma.spec.flags.env_gcc.enabled>
+# special handling bcverify.c for unexpected crash due to over-optimization in compilation
```

Comment:
```suggestion
# special handling bcverify.c to deal with ppcle64-specific crash when compiling with gcc 7.5
```

Reply:
Updated the comment against the suggestion above.